src/olympia/constants/promoted.py

```diff
@@ -0,0 +1,73 @@
+from django.utils.translation import ugettext_lazy as _
+
+
+class PromotedClass():
+    id = 0
+    name = 'Not Promoted'
+    search_ranking_bump = 0
+    warning = True
+    pre_review = False
+    admin_review = False
+
+
+class NOT_PROMOTED(PromotedClass):
+    pass
+
+
+class RECOMMENDED(PromotedClass):
+    id = 1
+    name = _('Recommended')
+    search_ranking_bump = 1000  # TODO: confirm this bump
+    warning = False
+    pre_review = True
+
+
+class VERIFIED_ONE(PromotedClass):
+    id = 2
+    name = _('Verified - Tier 1')
+    search_ranking_bump = 500  # TODO: confirm this bump
```

Comment:
Note: the current naive way we boost works ok for the limited number of extensions we need to boost up - worst case scenario we end up boosting a couple recommended results that were at the end of the results, almost irrelevant, but if we increase the number of things we boost, we might end up having to redesign the way it works to avoid littering the results with mostly irrelevant but verified/recommended extensions.